src/_utilities/helpers/helpers.md

```diff
@@ -81,3 +81,26 @@ Vi har også to samle klasser for *top-bottom* og *right-left*.
 <div class="margin-l--tb padding-l--tb"></div>
 <div class="margin-l--rl padding-l--rl"></div>
 ```
+
+## Border-radius
+
+Alle border-radius
+```html
+<div class="border-radius-s"></div>
+<div class="border-radius"></div>
+<div class="border-radius-l"></div>
+```
+
+Top border-radius
+```html
+<div class="border-radius-s--t"></div>
+<div class="border-radius--t"></div>
+<div class="border-radius-l--t"></div>
+```
+
+Bottom border-radius
+```html
+<div class="border-radius-s--b"></div>
+<div class="border-radius--b"></div>
+<div class="border-radius-l--b"></div>
```

Comment:
Og her

Reply:
Det gik vidst lige en tand for hurtigt, glemt .md filen.
merge master = "Already up to date".

Reply:
Der er vist stadig nogle merge-konflikter i border.css og helpers.md